schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1014 @@
+    },
+    "dateNotRecoveredOrDerecovered": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "dateRecovered": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "originalCaseDecisionDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "targetDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "appellantCommentsSubmitted": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "appellantStatementSubmitted": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "finalCommentsDueDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "interestedPartyRepsDueDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "lpaCommentsSubmitted": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "lpaProofsSubmitted": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "lpaStatementSubmitted": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "proofsOfEvidenceDueDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "siteNoticesSent": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "statementDueDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "$comment": "S78"
+    },
+    "inspectorNeedToEnterSite": {
+      "type": ["boolean", "null"],
+      "$comment": "S78"
+    },
+    "numberOfResidences": {
+      "type": ["number", "null"],
+      "description": "Number of residences involved in the appeal",
+      "$comment": "S78"
+    },
+    "siteGridReferenceEasting": {
+      "type": ["string", "null"],
+      "$comment": "S78"
+    },
+    "siteGridReferenceNorthing": {
+      "type": ["string", "null"],
+      "$comment": "S78"
+    },
```

Comment:
Aren't these integers? or do the come in another format?

Reply:
No idea, I followed the requirements in the spreadsheet... I understand these are used in Applications to generate location maps, which is a feature haven't heard of (yet) in S78 designs.

Reply:
OK we want to check that then, they are integers for NSIPs: https://github.com/Planning-Inspectorate/data-model/blob/main/schemas/nsip-project.schema.json#L220-L229

Reply:
I've changed this to integers, as it makes sense.